apps/challenges/views.py

```diff
@@ -1658,3 +1664,143 @@ def get_challenge_phase_by_pk(request, pk):
     )
     response_data = serializer.data
     return Response(response_data, status=status.HTTP_200_OK)
+
+
+@api_view(["POST"])
+@throttle_classes([UserRateThrottle])
+@permission_classes((permissions.IsAuthenticated, HasVerifiedEmail))
+@authentication_classes((ExpiringTokenAuthentication,))
+def invite_users_to_challenge(request, challenge_pk):
+
+    challenge = get_challenge_model(challenge_pk)
+
+    if not challenge.is_active or not challenge.approved_by_admin:
+        response_data = {"error": "Sorry, the challenge is not active"}
+        return Response(response_data, status=status.HTTP_403_FORBIDDEN)
+
+    try:
+        challenge_host = ChallengeHost.objects.get(user=request.user)
+    except ChallengeHost.DoesNotExist:
+        response_data = {"error": "You're not a challenge host"}
+        return Response(response_data, status=status.HTTP_400_BAD_REQUEST)
+
+    if not is_user_a_host_of_challenge(request.user, challenge.pk):
+        response_data = {
+            "error": "You're not authorized to invite a user in {}".format(
+                challenge.title
+            )
+        }
+        return Response(response_data, status=status.HTTP_400_BAD_REQUEST)
+
+    users_email = request.data.get("emails")
+
+    if not users_email:
+        response_data = {"error": "Users email can't be blank."}
+        return Response(response_data, status=status.HTTP_400_BAD_REQUEST)
+
+    try:
+        users_email = eval(users_email)
+    except Exception:
+        response_data = {"error": "Invalid format for users email."}
```

Comment:
To be consistent, can we please remove the period at the end of the error message?